src/test/java/integration/ShipmentControllerIT.java

```diff
@@ -342,12 +346,12 @@ public void removeShipmentGroup() throws Exception {
         testHelper.deleteShipment(shipment);
     }
 
-    private void checkDatesModifierAndCreator(long timeStarted, long timeFinished, Shipment createdShipment) {
-        long timeCreated = createdShipment.getCreated().getTime();
-        long timeModified = createdShipment.getLastModified().getTime();
+    private void checkDatesModifierAndCreator(LocalDateTime timeStarted, LocalDateTime timeFinished, Shipment createdShipment) {
+        LocalDateTime timeCreated = createdShipment.getCreated();
+        LocalDateTime timeModified = createdShipment.getLastModified();
 
-        assertTrue(WRONG_CREATED_MESSAGE, timeFinished >= timeCreated && timeCreated >= timeStarted);
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_CREATED_MESSAGE, timeFinished.isAfter(timeCreated) && timeCreated.isAfter(timeStarted));
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).